FWCore/Services/src/SiteLocalConfigService.cc

```diff
@@ -26,9 +26,36 @@ using namespace xercesc;
 //<<<<<< MEMBER FUNCTION DEFINITIONS                                    >>>>>>
 
 namespace {
+
+  template< class CharType >
```

Comment:
there is also Xerces.cc file right (in src directory). I would avoid creating a package just for this. It is not only header file but also the xerces-c dependency exported via FWCore/Concurrency/BuildFile.xml for the dependent packages